bin/hdfcoinc/pycbc_page_snrratehist

```diff
@@ -95,6 +95,38 @@ if not args.closed_box:
 
 if fstat is not None and not args.closed_box:
     le, re = bins[:-1], bins[1:]
+    # We need to plot a histogram "errorbar" with the hierarchically
+    # removed foreground triggers in red.
+    h_iterations = f['hierarchical_removal_iterations'][0]
```

Comment:
Is this backwards compatible with a file that does not have ``hierarchical_removal_iterations`` set? If not, make it backwards compatible.